index.html

```diff
@@ -24,9 +24,8 @@
         <div class="pages">
             <a href="/services">Services</a>
             <a href="https://twitter.com/pradumna_saraf">Twitter</a>
-            <a href="https://blog.pradumnasaraf.co">Twitter</a>
+            <a href="https://blog.pradumnasaraf.co">HashNode</a>
```

Comment:
```suggestion
            <a href="https://blog.pradumnasaraf.co">Blog</a>
```

Reply:
Thank you for the PR.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient